packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -725,9 +728,7 @@ class _RenderDecoration extends RenderBox
        _expands = expands,
        _material3 = material3;
 
-  // TODO(bleroux): consider defining this value as a Material token and making it
-  // configurable by InputDecorationThemeData.
```

Comment:
@bleroux Does this PR satisfy what you wanted from this TODO? Should we leave the part about defining it as a Material token?

Reply:
Oh I think you addressed this in your comment https://github.com/flutter/flutter/pull/175092#issuecomment-3280280915.